src/app/ui/button/button.ts

```diff
@@ -14,10 +14,15 @@ export class Button {
         const button = document.createElement('a');
         button.text = text;
         button.classList.add('gdg-raised-button');
+        button.classList.add('disabled');
         this.button = button;
         this.elem = button;
     }
 
+    ready():void{
```

Comment:
`ready` should be renamed to `enable` to match what it does
Otherwise you could create a `disabled` property with a setter that properly sets internal state